### DISCUSSION TITLE
Fastlane integration with Bitrise CI

### DISCUSSION THREAD

**Discussion by Koutenshidatshu** [fastlane/fastlane#17956] [Q&A]
I want to ask, i had Bitrise workflow with fastlane, 
The idea is when i start manually build configuration in bitrise, add some message (BITRISE_GIT_MESSAGE)
And the fastlane.file can read the message for firebase distribution release notes
This is possible to configuration in Fastlane file ?

**Comment by rogerluan** [comment]
Hey @Koutenshidatshu 👋 

Sounds like you're trying to read an environment variable in Fastfile. Fastfile is written in ruby, so you can read your env var as:

```ruby
git_message = ENV['BITRISE_GIT_MESSAGE']
```

Does this answer your question?
